src/reporting.h

```diff
@@ -22,5 +23,17 @@ void PrintReport(const snowcrash::Report& report,
                  const std::string& source,
                  const bool isUseLineNumbers);
 
+/**
+ *  \brief Print parser report to stderr.
+ *
+ *  \param report A parser report to print
+ *  \param source Source data
+ *  \param isUseLineNumbers True if the annotations needs to be printed by line and column number
```

Comment:
I think the `is` here feels misplaced. `useLineNumbers` would be clear on what the name is and I think I'd prefer the name `useLineNumbers`.